content/chainguard/chainguard-images/chainguard-registry/pull-through-guides/harbor/index.md

```diff
@@ -0,0 +1,132 @@
+5. Click **OK** to create the endpoint.
+
+After creating the endpoint, you can move on to creating a proxy cache.
+
+
+## Create a Proxy Cache
+
+A [*proxy* cache](https://goharbor.io/docs/2.1.0/administration/configure-proxy-cache/) allows Harbor to proxy and cache images from the Chainguard registry.
+
+To configure a cache, perform the following steps in the Harbor UI:
+
+1. Navigate to **Projects**.
+2. Click the **+ NEW PROJECT** button.
+3. Create the project with these details:
+    * **Project Name** — This can be whatever you like. This guide uses the name `cgr—proxy`.
+    * **Proxy Cache** — Toggle this on.
+    * **Endpoint** — Choose the endpoint you created in the previous step.
+4. Click **OK**
+
+Following that, you can pull images from the Harbor project like so:
+
+```shell
+docker pull $HARBOR_URL/cgr-proxy/$ORGANIZATION/$IMAGE:$TAG
+```
+
+Be sure to replace the placeholder values (`$HARBOR_URL`, `$ORGANIZATION`, `$IMAGE`, and `$TAG`) to reflect your own setup.
+
+
+## Create a Replication Rule
+
+A [*replication rule*](https://goharbor.io/docs/2.1.0/administration/configuring-replication/create-replication-rules/) is an alternative approach to a proxy cache. This section outlines how to configure a replication rule that copies images from the Chainguard registry to a Harbor project.
+
+First, create a project. From the Harbor UI, navigate to **Projects** and click the **+ NEW PROJECT** button.
+
+Give the project a name (this guide uses the name `cgr-mirror`) and be sure to leave the **Proxy Cache** option switched off.
+
+Then, perform the following steps to create a new replication rule:
+
+1. Navigate to **Administration > Replications**.
+2. Click the **+ NEW REPLICATION RULE** button.
+3. Create a new rule with these details:
+    * **Name** — This can be whatever you like. This guide uses the name  `cgr-mirror`.
+    * **Replication mode** — Must be `Pull-based`.
+    * **Source registry** — Specify the `cgr.dev` endpoint.
+    * **Source resource filter > Name** — (Optional) You can ensure you only select images in your organization by setting this to `<org-name>/*`.
+    * **Destination > Namespace** — This should be the project you created before.
+    * **Destination > Flattening** — Set to `Flatten All Levels`. This removes the organization name from the path.
+    * **Trigger Mode** — Set to `Scheduled` to run the replication regularly or `Manual` to trigger on an ad hoc basis.
+    * **Override** — Ensure this is enabled so that tags in the destination project are replaced when they change.
+4. Click the **Save** button.
+
+To trigger the replication manually, select the `cgr-mirror` rule in the table and click the **REPLICATE** button. Then, navigate to **Projects > cgr-mirror** and observe images populating.
+
+You should be able to pull images from the project like this:
+
+```shell
+docker pull $HARBOR_URL/cgr-mirror/$ORGANIZATION/$IMAGE:$TAG
```

Comment:
this is wrong  -- if set to "flatten all" the $ORG is removed i.e:

docker pull $HARBOR_URL/cgr-mirror/$IMAGE:$TAG